packages/shared/src/components/profile/Header.tsx

```diff
@@ -93,19 +87,27 @@ export function Header({
           target={TargetId.MyProfile}
         />
       )}
-      <Button
-        variant={ButtonVariant.Float}
-        size={ButtonSize.Small}
-        icon={<ShareIcon />}
-        onClick={() => onShareOrCopyLink()}
-      />
       <FollowButton
         entityId={user.id}
         type={ContentPreferenceType.User}
         status={contentPreference?.status}
         entityName={`@${user.username}`}
         className="ml-2 flex-row-reverse"
       />
+      {!isSameUser && (
+        <CustomFeedOptionsMenu
+          className="ml-2"
+          shareProps={{
+            text: `Check out ${user.name}'s profile on daily.dev`,
+            link: user.permalink,
+            cid: ReferralCampaignKey.ShareProfile,
+            logObject: () => ({
+              event_name: 'share profile',
```

Comment:
This should be a enum (we have a list already)

Reply:
@rebelchris Copy pasted this from the existing one. I assume you are referring to the ReferralCampaignKey enum? In that case the value is `share-profile` instead. Not sure if that matters?

Reply:
No all event_names should come from a LogEvent or something like that.
Only some old ones don't have it, but let's fix it.